PeteUgwu/To-Do-list
# Testing To Do list: Interactions

## Testing To Do list: Interactions

- Use jest framework for testing.
- Use already created mock object in the previous test to mock the localStorage updates.
- Mock the HTML elements to test DOM manipulation functions.
- Make sure you group your tests using the describe() method.
- Fix linters
